.github/ISSUE_TEMPLATE/REPORT_A_BUG.md

```diff
@@ -64,6 +64,13 @@ test.css
 2:4    Expected "11.50" to be one of "10, 11.5, 14" (scales/font-size)
 ```
 
+> Please request the `all-contributors` bot add you to the repo README as well, and thanks for helping out.
```

Comment:
perhaps link to how to do this https://allcontributors.org/docs/en/bot/usage
or inline the information?